DATA/UNIVERSE/SYSTEMS/Br04/Br04.ini

```diff
@@ -4555,12 +4884,79 @@ archetype = space_industrial01
 reputation = br_m_grp
 parent = Br04_02
 
+[Object]
+nickname = Br04_docking_fixture_1
+ids_name = 261166
+pos = 13159, 350, 35847
+rotate = 0, -105, 0
+archetype = docking_fixture
+ids_info = 66489
+reputation = br_p_grp
+behavior = NOTHING
+dock_with = Br04_01_base
+base = Br04_01_base
+
+[Object]
+nickname = Br04_to_Br05_holeb
+ids_name = 260644
+pos = -44140, 0, 37485
+archetype = jumphole_red
+msg_id_prefix = gcs_refer_system_Br05
+jump_effect = jump_effect_hole
+ids_info = 66146
+visit = 0
+goto = Br05, Br05_to_Br04_holeb, gate_tunnel_bretonia
+
+[Object]
+nickname = Br04_07
+ids_name = 196680
+;res str
+; Maghera Outpost
+pos = -49531, 0, 57324
+rotate = 0, 0, 0
+archetype = space_port_dmg
+ids_info = 65673
+;res html
+; Maghera Outpost is an old abandoned cloud reprocessing factory left behind by the Bretonian government in 732 AS. Due to many public protests over the environmental disaster in Leeds, the government launched a program to attempt to turn the harmful clouds into harmless, and potentially useful, gasses that could be used as a low-grade fuel source. After eight years the prime minister ended the program for lack of any useful progress.
+; 
+; In recent years the Mollys discovered this outpost lost in a radiation zone in the West Leeds cloud. They have since converted it into a ship reassembly plant where they produce their newest ship: the Foxhound.
+; 
+; Maghera also serves as a staging point for all of the Molly attacks in Leeds, including LD-14 for MOX and Stokes for Niobium, and throughout Bretonian space. So far the government hasn't caught on.
+base = Br04_07_Base
+dock_with = Br04_07_Base
+reputation = fc_m_grp
+behavior = NOTHING
+voice = atc_leg_f01a
+space_costume = , robot_body_B

```

Comment:
Want to keep it with a robot, or maybe adding a Molly guy here? I can live with both really.